### PR TITLE
Legacy build: make coqide errors when coqide configured off

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

File: Makefile.ide [hunk 1]
```diff
@@ -30,6 +30,8 @@ $(IDETOP): $(ALL_ML_SOURCE_FILES) $(PROTO_IDE_FILES) ide/coqide/idetop.ml
 
 ifeq ($(HASCOQIDE),no)
 coqide:
+	>&2 echo "Coqide compilation configured off."
```
Comment (Alizter) [inline Makefile.ide:33]:
```suggestion
	>&2 echo "CoqIDE compilation configured off."
```

**Review by Alizter — APPROVED** [review]

File: Makefile.build [hunk 2]
```diff
@@ -88,7 +88,7 @@ include Makefile.dev
 
 .PHONY: world byte coq world.timing.diff coq.timing.diff states
 
-world: coq coqide documentation revision
+world: coq $(if $(filter-out no,$(HASCOQIDE)),coqide,) documentation revision
```
Comment (gares) [inline Makefile.build:91]: It would a bit more verbose, but easier to read, if you use a toplevel if as in the other file

**Comment by Alizter** [comment]
merging soon

**Comment by Alizter** [comment]
@coqbot merge now

**Pull request merged**